src/quickpicks/topics.ts

```diff
@@ -0,0 +1,43 @@
+import { ProgressOptions, ThemeColor, ThemeIcon, window } from "vscode";
+import { KafkaTopic } from "../models/topic";
+import { getSchemasForTopicEnv, getTopicsForCluster } from "../viewProviders/topics";
+import { KafkaCluster } from "../models/kafkaCluster";
+import { IconNames } from "../constants";
+
+export async function topicQuickPick(cluster: KafkaCluster): Promise<KafkaTopic | undefined> {
+  const options: ProgressOptions = {
+    location: { viewId: "confluent-topics" },
+    title: "Loading topics...",
+  };
+  return window.withProgress(options, async () => {
+    const topics: KafkaTopic[] = await getTopicsForCluster(cluster, true);
```

Comment:
Why force a deep-fetch round trip here (the `true`) param? Why not allow cached topics to be used if known?

Reply:
Do we have something in place that updates list of topics occasionally? Not familiar with the part.

Reply:
Per zoom convo: we don't have anything automatic, but user can trigger these updates via other interactions with tree views.

Reply:
Per zoom convo: we explicitly choose to show the most up-to-date data here, since the command can be used without interaction with the rest of extension UI.